test/powershell/Language/Parser/TypeAccelerator.Tests.ps1

```diff
@@ -1,17 +1,411 @@
 
 Describe "Type accelerators" -Tags "CI" {
-    $TypeAcceleratorsType = [psobject].Assembly.GetType("System.Management.Automation.TypeAccelerators")
+    BeforeAll {
+        
+        $TypeAcceleratorsType = [psobject].Assembly.GetType("System.Management.Automation.TypeAccelerators")
+        
+        $TypeAccelerators = $TypeAcceleratorsType::Get
+        $TypeAcceleratorsType::Add('msft_2174855', [int])
+        $TypeAcceleratorsType::Add('msft_2174855_rm', [int])

```

Comment:
it would good to decode 'msft_2174855' and 'msft_2174855_rm' - can we use more informative names and comments for it?